resources/hotels.py

```diff
@@ -0,0 +1,96 @@
+from flask import abort
+from flask_restful import Resource, reqparse, fields, marshal
+
+# TODO: create a postgres database and store these there
+hotels = [
+    {
+    'id': 1,
+    'hotel_name': 'The Asteroid',
+    'total_double_rooms': 10,
+    'total_queen_rooms': 10,
+    'total_king_rooms': 10,
+    'created_date': '2020-06-03'
+    }
+]
+
+hotel_fields = {
+    'id': fields.Integer,
+    'hotel_name': fields.String,
+    'total_double_rooms': fields.Integer,
+    'total_queen_rooms': fields.Integer,
+    'total_king_rooms': fields.Integer,
+    'created_date' : fields.String,
```

Comment:
should we consider using a `Date` type for this?

Reply:
The fields.DateTime field type will just return a iso formatted string in the json object anyway, so I'm thinking we don't need it unless we pass it date objects. It may be easier to just deal with strings for now.